### ISSUE TITLE
Feature Request: Add compatibility with Volar

### ISSUE THREAD

**Issue by MatthewAry** [mxsdev/ts-type-explorer#43]
[Volar](https://volarjs.dev/) is used by Vue, Svelte, and Astro to help bridge the gap between Typescript and their own file types. [Volar is currently undergoing a rewrite at the moment](https://github.com/volarjs/volar.js/issues/85), once they're done, I hope to see this extension support these incredible ecosystems.

**Comment by zcf0508** [comment]
I submit a pr to get type from `vue` file. You can check out #45 .

**Comment by mrleblanc101** [comment]
Yes, currently this only work inside `.ts` files, and not inside `<script lang="ts">` of Vue Single File Component.
I was facing the same issue, this would indeed be a welcome improvement !

**Comment by Anoesj** [comment]
Yes, this would make the lives of so many devs much easier!

Note that Volar's v2 has been released in Jan 2024: https://github.com/volarjs/volar.js/releases/tag/v2.0.0
They've done numerous releases since then and are at v2.4.13 at the time of writing, meaning it's definitely stable enough to build upon at this point.
